pageserver/src/basebackup.rs

```diff
@@ -139,6 +139,8 @@ where
     async fn send_tarball(mut self) -> anyhow::Result<()> {
         // TODO include checksum
 
+        let on_demand_slru_download = !self.full_backup; // TODO: should it be feature flag, config parameter or whatever else ?
+
```

Comment:
Good question. I think this needs to be deployed in two stages: the compute changes first, and only after all new computes are guaranteed to support the on-demand download, change the pageserver to omit the SLRUs from the tarball.

Perhaps the easiest way to do that is to have `on_demand_slru_download = false` here for now, and get this PR committed that way. Later, as a separate PR change that to `on_demand_slru_download = !self.full_backup`.

Do we need to worry about existing long-running computes, being restarted, and performing new basebackups later but still running with old binaries?

Reply:
Done: I disable for a while downloading SLRU segment on demand in PS.

Reply:
I added `lazy_slru_download` tenant config option to page server and performance test for startup with lazy SLRU downloads